python-main.js

```diff
@@ -467,7 +467,7 @@ function web_editor(config) {
                         scaleSpeed: zoomScaleSteps + 1.0
                     }
                 });
-                function myUpdateFunction(event) {
+                var x = function myUpdateFunction(event) {
```

Comment:
Did this work correctly? I was expecting something more along the lines of assigning an anonymous function to a variable named `myUpdateFunction`:

```
var myUpdateFunction = function(event) {
```